aspnet/security/accconfirm.rst

```diff
@@ -1,236 +1,209 @@
+To complete this section, you must first enable an external authentication provider. See :doc:`sociallogins`.
 
-.. image:: accconfirm/_static/loginaccconfirm5.PNG
+You can combine local and social accounts by clicking on your email link. In the following sequence "RickAndMSFT@gmail.com" is first created as a local login, but you can create the account as a social log in first, then add a local login.
 
-- After your password has been successfully reset, you can login with your email and new password.
+.. image:: accconfirm/_static/rick.png
 
-.. image:: accconfirm/_static/loginaccconfirm6.PNG
+Click on the **Manage** link. Note the 0 external (social logins) associated with this account.
 
-         
-Require email confirmation before login
----------------------------------------
-Currently once a user completes the registration form, they are logged in. You generally want to confirm their email before logging them in. In the section below, we will modify the code to require new users to have a confirmed email before they are logged in (authenticated). Update the HttpPost Login action with the following highlighted changes.
+.. image:: accconfirm/_static/manage.png
 
-.. code-block:: c#
+Click the link to another log in service and accept the app requests. In the image below, Facebook is the external authentication provider:
+
+.. image:: accconfirm/_static/fb.png
 
-    public async Task<IActionResult> Login(LoginViewModel model, string returnUrl = null)
-    {
-        ViewBag.ReturnUrl = returnUrl;
-        if (ModelState.IsValid)
-        {
-            // Require the user to have a confirmed email before they can log on.
-            var user = await UserManager.FindByNameAsync(model.Email);
-            if (user != null)
-            {
-                if (!await UserManager.IsEmailConfirmedAsync(user))
-                {
-                    ModelState.AddModelError(string.Empty, "You must have a confirmed email to log in.");
-                    return View(model);
-                }
-            }
-            // Code removed for brevity. You should have the code that was in the project.
-        }
-
-        // If we got this far, something failed, redisplay form
-        return View(model);
-    }
-
-
-
-Next steps
-----------
-- Once you publish your Web site to Azure Web App, you should reset the secrets for SendGrid. Set the SendGrid secrets as application settings in the Azure Web App portal. The configuration system is setup to read keys from environment variables.
-
-Summary
--------
-
-ASP.NET Identity can be used to add account confirmation and password recovery.
+The two accounts have been combined, you will be able to log on with either account. You might want your users to add local accounts in case their social log in authentication service is down, or more likely they have lost access to their social account. 
```

Comment:
Recommend adding a Summary that ties everything up that the reader learned in this article. Something like, "In this article, you learned ..."